newslet/surveys/views.py

```diff
@@ -1,4 +1,8 @@
 from django.shortcuts import render
+from website.models import GuestSurvey
+from django.contrib.auth.decorators import login_required
+
+
 
```

Comment:
mam wrazenie ze ten plik to jest jakas stara wersja kodu albo dziwne ze mi sie nie wyswietla, bo brakuje mi tutaj widokow ktore dodawalem: CompanyCreateView, CompanyListView, ProductCreateView, ProductListView, SurveyCreateView

Reply:
to jest faktycznie robione ze starego brancha, ale jak sie zmerguje to powinno byc git majonez :)